CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 0.4.0
+
+* Added an extension method on `String` to allow easy access to the `Characters`
+  of the string:
+
+  ```dart
+  print('The first character is: ' + myString.characters.take(1))
```

Comment:
Should be `.first` instead of `.take(1)`. The latter returns a `Characters`, not a `String`, and `Characters` cannot be added to a string. 
(Would work if you use string interpolation instead of `+` because the `Characters` `toString` returns the string).